### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by grandoc**

### CODE REVIEW

File: htdocs/comm/mailing/class/html.formadvtargetemailing.class.php [hunk 1]
```diff
@@ -16,9 +16,9 @@
  */
 
 /**
- * \file    comm/mailing/class/html.formadvtragetemaling.class.php
+ * \file    comm/mailing/class/html.formadvtragetemailing.class.php
```
Comment (frederic34) [inline htdocs/comm/mailing/class/html.formadvtargetemailing.class.php]: formadvtragetemailing -> formadvtargetemailing

Comment (grandoc) [reply htdocs/comm/mailing/class/html.formadvtargetemailing.class.php]: indeed! I was focus on mailing...

**Comment by grandoc** [comment]
I just don't see what can be wrong ?

**Comment by frederic34** [comment]
https://github.com/Dolibarr/dolibarr/pull/9260

**Pull request merged**